### ISSUE TITLE
Opening books stored on obsolete repository

### ISSUE THREAD

**Issue by Stefano80** [official-stockfish/Stockfish#1417]
I wanted to do some work on this https://github.com/glinscott/fishtest/issues/139

and I discovered that the opening books are stored on a very old repository

https://github.com/mcostalba/FishCooking/blob/setup/8moves_v3.pgn

We should move them somewhere where they can be maintained. The question is: fishtest or stockfish repository? Any idea?

**Comment by snicolet** [comment]
Fishtest, I would say. The books are mainly used in the framework, and will probably never end as part of the Stockfish distrib.

**Comment by Stefano80** [comment]
Ok, I will let it open for some time and if also @ppigazzini agrees, I will move everything over there.

**Comment by ppigazzini** [comment]
I agree to move the books and the cutechess-cli binaries (I routinely compile [up to date and static versions of cutechess-cli](https://github.com/ianfab/FishCooking) for multi variant fishtest, in official fishtest an up to date version is required to run fishtest w/ msys2/cygwin python).
Obviously the [fishtest code](https://github.com/glinscott/fishtest/blob/master/worker/games.py#L43) must be updated to reflect the change of repository.

**Comment by Stefano80** [comment]
Ok. Then will do as soon as I learn how (given that Pasquale does not preempt me). We will change the code after that we have all stuff in fishtest repo and then @mcostalba can remove it from its repo (if he wishes so).

**Comment by Stefano80** [comment]
Ah I just found out that the files are just added to the repository. I thought there is something special about them, I am so stupid some time.

**Comment by mcostalba** [comment]
Books are not good opening books, i.e. they are not meant for turnaments. They are good for testing becuase are specifically designed to increase engine resolution (they are somewhat unbalanced so that you can spot differences testing two almost identical engines), but not good for playing.

**Comment by ppigazzini** [comment]
Just as information: @ianfab had coded a [book generator](https://github.com/ianfab/bookgen) that follow the [recipe](https://groups.google.com/forum/#!msg/fishcooking/pt9MBSQMZs0/omErNOzyxesJ) used for the SF 2 moves book.

**Comment by snicolet** [comment]
@Stefano80 @ppigazzini @mcostalba 

What is the status of this issue? I understand that there is a consensus that it could be useful if the books could be moves to the official fishtest repository, along with up to date and static versions of cutechess-cli, but I don't know if we have done that already.

**Comment by ppigazzini** [comment]
@snicolet Some days ago I created a branch on current fishtest repo (view this PR https://github.com/glinscott/fishtest/pull/240) to have openings books and cutechess-cli binaries. This slowed down the git clone operations used with the dev-server and the test workers (to be sure to test the correct configuration I delete and then clone the fishtest repo) so I closed the PR and deleted the branch.
A solution could be to have a new Github account, e.g. official-fishtest, with all the repos required to run fishtest.
